### DISCUSSION TITLE
Name and branding

### DISCUSSION THREAD

**Discussion by brafdlog** [brafdlog/caspion#139] [General]
We should decide on a name for the app. I think this is a prerequisite to doing a wider release.
Once we have that it should be consistent across the codebase. This will also imply a logo.

@baruchiro Let's use this to brainstorm Ideas for names.

Something in the world of:
חיסכון, מעקב, שליטה בהוצאות

The name חזרזיר is stuck in my brain but its probably not a great one :)

**Comment by baruchiro** [comment]
How the חזרזיר is related to the world you described?

Anyway, my old app called עו"שי.

I don't really care about the name, I just want a unique-non general name, that's all.
I'm not good at naming.

**Reply by baruchiro** [reply]
I will ask some friends

**Comment by giladtaase** [comment]
חזרזיר כמו קופת חיסכון

**Reply by baruchiro** [reply]
אם כבר מתוחכם אז "פסחזון". מעניין שעוד אין חברה כזאת!

הבעיה היא שאנחנו לא באמת מדברים על חסכון, רק על "מה קורה לכסף שלי"

**Reply by giladtaase** [reply]
שם חמוד למרות שאף פעם לא הבנתי את קרת, אבל לכתוב אותו באנגלית זה סיוט

**Reply by baruchiro** [reply]
מזל שהקהל שלנו ישראלי...
אולי PassoverArea? PassoverZone? 😂😂😂

Pesachzon

**Reply by brafdlog** [reply]
כספיון קנה אותי. פשוט, רלוונטי ועם קונוטציות טובות

**Comment by brafdlog** [comment]
כספי?

**Comment by giladtaase** [comment]
כספיון :)

**Reply by brafdlog** [reply]
מעולה!

**Reply by baruchiro** [reply]
מה יש לכם מחיות אתם?

**Comment by Moshet** [comment]
מציע את "חיובי" - גם משחק על הקטע של חיובים כספיים וגם על הקטע של פלוס, מאזן וכו'

**Reply by baruchiro** [reply]
וואלה יפה!

**Comment by brafdlog** [comment]
גם חיובי וגם כספיון הם הם נוגעים לנושא ונותנים קונוטציה חיובית.

חסרון שאני רואה בחיובי הוא שיהיה לא ברור איך מבטאים את זה ואני לא בטוח לגבי הברנדינג שיצא מזה.

כספיון במובן הזה הוא מאוד ברור וגם קל להבין איזה לוגו וכו נוציא משם.

בוא נעשה הצבעה.

**Reply by baruchiro** [reply]
שמע, נראה שבסוף צדקת. קיבלנו כמה פידבקים לגבי זה שכשמכניסים "חיובי" במשפט נורא קשה להבין מה קורה.
חוצמיזה שגם קשה לחפש את המילה הזאת בגוגל, לכאורה.

האם זאת עילה מספיקה לשינוי שם שוב פעם?

**Reply by baruchiro** [reply]
אם כן, אולי כדאי לפתוח את ההצבעה ליותר אנשים?
או שאולי אפשר למשוך עוד קצת, לצבור יותר משתמשים ולשתף אותם?

**Comment by brafdlog** [comment]
הצבעה על שם לאפליקציה:

1. כספיון
2. חיובי
3. פסחזון

@giladtaase @baruchiro @barakd @udielenberg @Moshet @Arielgordon123

**Reply by brafdlog** [reply]
אני מצביע כספיון 👍🏾

**Reply by giladtaase** [reply]
כספיון

**Reply by Moshet** [reply]
אם התוכנה היא באנגלית אז מסכים לגבי אופן הביטוי של חיובי... אבל אם היא באנגלית, אז אולי כדאי שזה יהיה משהו קצת יותר בינלאומי.
מציע את InCharge
שם שמבטא אחריות, שליטה, חיובים...
בד"כ שמות עם הברה אחת או שתיים מקסימום הם יותר קליטים.

**Reply by giladtaase** [reply]
התוכנה היא לבנקים ישראליים, אז עדיף שם עברי, אבל בכל זאת חשוב לדעתי שאפשר יהיה לכתוב את השם באנגלית באופן ברור וחד משמעי.
InCharge נשמע לי מעולה לאפליקציה בינלאומית...
למעשה נראה לי שיש גוף כמו "פעמונים" בשם הזה בדיוק.
https://www.incharge.org/

**Reply by brafdlog** [reply]
אני מסכים שכיוון שהתוכנה היא לבנקים ומוסדות פיננסיים ישראליים הגיוני שיהיה לה שם בעברית. הממשק גם הולך להשתנות בעתיד לעברית כי זה לא באמת מעניין אף קהל אחר.

**Reply by baruchiro** [reply]
<div dir='rtl' align='right'>
כן, עברית...
אז @Moshet תצביע שוב

אני בעד **חיובי**, כי זה עושה קונוטציה מאוד חיובית.
לגבי ביטוי באנגלית- נו, ch, מה כבר יקרה?
ולגבי branding? לא מבין בזה, אבל סמיילי עם דולרים בעיניים, אני יודע מה...
</div>

**Reply by barakd** [reply]
אני חיובי על חיובי

**Reply by Moshet** [reply]
אני גם עם חיובי, במיוחד בגלל שזה יתרכז בקהל ישראלי:)

**Reply by brafdlog** [reply]
אז יש לנו שלוש שתיים לטובת חיובי. 

ניתן הזדמנות גם ל @udielenberg  להצביע ונחליט

**Reply by Arielgordon123** [reply]
אני גם בעד חיובי

**Comment by udielenberg** [comment]
@brafdlog 

טוב, ״כספיון״ הוא מעט משעשע
ו״חיובי״ הוא מאוד סטרייט פורוורד

ככה שזה מאוד תלוי בקהל המשתמשים.
אני אישית מעדיף את כספיון, אבל נראה לי ש״חיובי״ יקלע לקהל יותר רחב כי הוא גנרי.

מעבר לזה רעיון לא רע לתפוס נדלן על השם ״חיובי״ :) 

אז ההחלטה הבוגרת תהיה ״חיובי״

### **אני בעד ״חיובי״**

**Reply by baruchiro** [reply]
תודה על התשובה המפורטת, זה מוסיף בטחון ;-)

**Comment by brafdlog** [comment]
אוקיי, הרוב קובע **חיובי**

**Reply by giladtaase** [reply]
אז תגיד חיובי!

**Comment by baruchiro** [comment]
@brafdlog How to write it in English?

- positive
- Chiuvi
- Hiuvi

**Reply by brafdlog** [reply]
I would go with Hiuvi

**Reply by baruchiro** [reply]
OK so we are ready for #140 and #103

**Comment by brafdlog** [comment]
אני עדיין בעד כספיון, בין השאר מהסיבות שאמרת.
אפשר להחליף, אולי לפני שתפרסם פוסט נוסף.

**Reply by baruchiro** [reply]
אתה רוצה לשאול עוד אנשים, בפוסט או משהו, או שפשוט נזרום על זה?

יש לי איזה תחושה שעדיף כבר לזרום על זה בלי להסתבך עם עוד דיונים ואפשרויות

**Reply by brafdlog** [reply]
לא רואה צורך לדון בזה עוד

**Reply by baruchiro** [reply]
בזמנך החופשי :-)

**Reply by baruchiro** [reply]
שאין לך

**Reply by baruchiro** [reply]
Reopen: #140 #103
